.github/workflows/push.yml

```diff
@@ -26,12 +23,33 @@ jobs:
           poetry install
           poetry run pylint -E axiom tests
           poetry run black --check axiom tests
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+      - name: Install poetry
+        run: pip install poetry==1.1.7
+      - name: Run tests
+        env:
+          AXIOM_DEPLOYMENT_URL: ${{ secrets.AXIOM_DEPLOYMENT_URL }}
+          AXIOM_TOKEN: ${{ secrets.AXIOM_TOKEN }}
+          AXIOM_ORG_ID: ${{ secrets.AXIOM_ORG_ID }}
```

Comment:
This won't work on main, see axiom-go:

https://github.com/axiomhq/axiom-go/blob/610cfbd235d3df17f96a4bb156c50385cfbd9edd/.github/workflows/push.yml#L76-L77

You can leave the org id for cloud though, it should be ignored for self-host deployments

Reply:
updated